src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import SearchResults from "../../src/components/SearchResults";
+
+describe("SearchResults", () => {
+  
+    const validProp = {
+      results: `https://images-api.nasa.gov/search?q=moon`
+    }
+
+    test("check the component renders correctly", () => {
+    const rendered = render(<SearchResults results={validProp}/>);
+
+    expect(rendered).toMatchSnapshot(validProp.results);
```

Comment:
If you are doing a snapshot test, make sure to destructure out the fragment and test that against your snapshot, as you've done in the other tests